benefits/oauth/views.py

```diff
@@ -47,8 +48,49 @@ def authorize(request):
     if token is None:
         logger.warning("Could not authorize OAuth access token")
         return redirect(ROUTE_START)
-    else:
-        # we are intentionally not storing anything about the user, including their token
-        logger.debug("OAuth access token authorized")
-        session.update(request, auth=True)
-        return redirect(ROUTE_CONFIRM)
+
+    logger.debug("OAuth access token authorized")
+
+    # we store the id_token in the user's session
+    # this is the minimal amount of information needed later to log the user out
+    session.update(request, oauth_token=token["id_token"])
+
+    return redirect(ROUTE_CONFIRM)
+
+
+def logout(request):
+    """View implementing OIDC and application sign out."""
+
+    # overwrite the oauth session token, the user is signed out of the app
+    token = session.oauth_token(request)
+    session.update(request, oauth_token=False)
+
+    origin = session.origin(request)
+    redirect_uri = request.build_absolute_uri(origin)
```

Comment:
Rather than passing different endpoints to the auth server and having to ensure each is registered, what do you think about making a single logout callback endpoint, then redirect to the `origin` from there? This would mirror what we do on the login side.

Reply:
I really like this idea 👍

Reply:
Done in [`512c2f4`](https://github.com/cal-itp/benefits/pull/436/commits/512c2f47a14bb3ca26f083afad19afaa48f80f2e)